content/api/commands/clearcookie.md

```diff
@@ -39,6 +39,7 @@ Pass in an options object to change the default behavior of `cy.clearCookie()`.
 
 | Option    | Default                                                        | Description                                                                              |
 | --------- | -------------------------------------------------------------- | ---------------------------------------------------------------------------------------- |
+| `domain`  | Superdomain of the current URL                                 | Clears the cookie from the specified domain                                              |
```

Comment:
should we link the user to what a `superdomain` is?

Reply:
Maybe once there's one to link to as part of https://github.com/cypress-io/cypress-documentation/pull/4809? 😬

Reply:
That would be ideal. Gonna see if I can look into that soon so we at least have a reference